notify_login_email_changed/models/res_partner.py

```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8 -*-
+# Copyright 2020 Quartile Limited
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+from odoo import models
+
+
+class ResPartner(models.Model):
+    _inherit = "res.partner"
+
+    def _send_email(self, mail_template, old_email="", new_email=""):
+        self.ensure_one()
+        ctx = {
+            "default_composition_mode": "comment",
+            "mail_create_nosubscribe": True,
+            "disable_message_subscribe": True,
+            "template_type": "comment",
+            "old_email": old_email,
+            "new_email": new_email,
+        }
+        template = mail_template.with_context(ctx)
+        template.send_mail(self.id, force_send=True)
+        return True
+
+    def write(self, vals):
```

Comment:
Can we put the logic to the controller side, i.e. '/my/account'?
Since this is a feature for the portal users.

Reply:
I updated